pages/vi/nextcloud-tor.md

```diff
@@ -12,7 +12,7 @@
 
 ## Prepare Treehouses image
 
-Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
+Download the latest treehouses image from https://treehouses.io/#!pages/download.md, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
```

Comment:
@law-lin  please use a relative link instead of an absolute one?

Reply:
@dogi Just pushed an update - please review. Here's the raw.githack link:

http://raw.githack.com/law-lin-OLE/law-lin-OLE.github.io/nextcloud-tor-download-link/#!pages/vi/nextcloud-tor.md